docs/en/sql-reference/table-functions/mongodb.md

```diff
@@ -0,0 +1,74 @@
+---
+slug: /en/sql-reference/table-functions/mongodb
+sidebar_position: 42
+sidebar_label: mongodb
+---
+
+# mongodb
+
+Allows `SELECT` queries to be performed on data that is stored on a remote MongoDB server.
+
+**Syntax**
+
+``` sql
+mongodb(host:port, database, collection, user, password [, options])
```

Comment:
`structure` probably should be added here, because this argument is required, not optional as in other table functions